Service/RequestService.php

```diff
@@ -267,7 +267,11 @@ public function proxyHandler(array $data, array $configuration, ?Source $proxy =
 
         // Get clean query paramters without all the symfony shizzle.
         $query = $this->realRequestQueryAll($this->data['method']);
-        $this->data['path'] = '/'.$data['path']['{route}'];
+        if(isset($data['path']['{route}'])) {
```

Comment:
eigenlijk moet hier dan dus een === true in maar ok

Reply:
prima, pas ik die nog aan